.github/workflows/main.yml

```diff
@@ -48,23 +48,22 @@ jobs:
     steps:
 
       - name: Checkout main Joplin repository
-        uses: actions/checkout@v2
+        uses: actions/checkout@v4
         with:
           repository: 'laurent22/joplin'
           ref: dev
           ssh-key: ${{ secrets.JOPLIN_BOT_SSH_PRIVATE_KEY }}
           path: 'joplin'
 
       - name: Checkout Joplin website repository
-        uses: actions/checkout@v2
+        uses: actions/checkout@v4
         with:
           repository: 'joplin/website'
           ref: master
-          ssh-key: ${{ secrets.JOPLIN_BOT_SSH_PRIVATE_KEY }}
```

Comment:
Cloning `joplin/website` with an `ssh-key` should no longer be necessary &mdash; that repository does not need to be updated by this script.